tests/test_recipe_synth_basics.py

```
@@ -453,3 +454,201 @@ def test_synthesizer_default_templates_structure(self):
+
+        recipe = synthesizer._create_recipe_from_template(
+            template=template, ingredients=ingredients, servings=2
+        )
+
+        assert isinstance(recipe, Recipe)
+        assert recipe.servings == 2
+        assert len(recipe.ingredients) == len(ingredients)
+
+    def test_generate_recipe_name(self, synthesizer):
+        """Test _generate_recipe_name method."""
+        template = list(synthesizer.templates.values())[0]
```

Ruff (0.14.1)

Prefer `next(iter(synthesizer.templates.values()))` over single element slice

Replace with `next(iter(synthesizer.templates.values()))`

(RUF015)